static/js/eclipsefdn.adopters.js

```diff
@@ -65,6 +65,7 @@
     console.log(opts)
```

Comment:
Is this needed?

Reply:
Ah, that was there before. I can remove it. @autumnfound I think we don't need this right?

Reply:
No, you can remove it, it was probably forgotten

Reply:
this is going to be removed in https://github.com/EclipseFdn/solstice-assets/pull/209/files

Reply:
I don't think I should include static/ for PR? It should be generating the correct static when build? 🤔

Reply:
Typically yeah. But didn't you add code to static?

Reply:
or was that from the build?

Reply:
what do you mean by "add code to static"? I think it's from my local build

Reply:
Actually, let me check, I feel this PR also include old commits. I already forgot what happened last year ...

Reply:
Ah, I think it's because the Hugo master has not included the newest assets, which is already fixed:
https://github.com/EclipseFdn/solstice-assets/blob/master/js/eclipsefdn.adopters.js

That's why it shows on my local build

Reply:
Yep, not much you can do for this file.